sdk/cosmos/azure-cosmos-benchmark/src/test/java/com/azure/cosmos/benchmark/ReadMyWritesConsistencyTest.java

```diff
@@ -132,6 +132,7 @@ public void readMyWrites(boolean useNameLink) throws Exception {
         ReadMyWriteWorkflow wf = new ReadMyWriteWorkflow(cfg) {
             @Override
             protected void onError(Throwable throwable) {
+                logger.error("Error occurred in ReadMyWriteWorkflow", throwable);
```

Comment:
Will error level enabled check come later?

Reply:
Since this is logging error in tests, I don't think if we need error level enabled check for this. 
This is only for debugging CI

Reply:
slf4j internally will check the log level, so checking log level is not needed in this case.

checking log level is needed only when the information needed to be logged requires some pre-computation. For example in the following case checking log level helps:

```java
// checking log level helps
if (logger.isDebugEnabled()) {
 String diagnosticsInfoAsString = diagnosticsInfo.toJson()
 logger.debug("diagnostics info is {}, diagnosticsInfoAsString);
}
```
but here it is not needed
```java
// checking log level is not needed as no pre-computation is done here. slf4 will do internal checking.
 logger.debug("endpoint is {}, endpoint);
}
```

Reply:
Agreed, thanks for the reference @moderakh